data/strings/strings.txt

```diff
@@ -15968,50 +15968,98 @@
     zh-Hans = 截至 %@ 的社区创建的 OpenStreetMap 数据。请访问 OpenStreetMap.org 以了解有关如何编辑和更新地图的信息。
     zh-Hant = 截至 %@ 的社群創建的 OpenStreetMap 資料。請訪問 OpenStreetMap.org 以了解有關如何編輯和更新地圖的更多資訊。
 
+  [login_osm_presentation]
+    comment = OSM explanation on Android login screen
+    tags = android
+    en = OpenStreetMap.org (OSM) is a community project to build a free and open map. It's the main source of map data in Organic Maps and works similar to Wikipedia. You can add or edit places and they become available to millions of users all over the World.\nJoin the community and help to make a better map for everyone!
```

Comment:
```suggestion
    en = OpenStreetMap.org (OSM) is an open community project to build a free map. It's the main source of map data in Organic Maps and works similarly to Wikipedia. You can add or edit places and they become available to millions of users all over the World.\nJoin the community and help to make a better map for everyone!
```

Reply:
I didn't mean "open" in the sense that any one can join, that's covered with "community", but "open" in the sense of open data.

Reply:
yeap, "free and open map" sounds better to me than just "free map" (sounds like gratis is the main property of it..)

Reply:
We may also use "libre" word to avoid confusion of "open/free".

Reply:
I wouldn't use "libre" as too many people don't know what it means.